### PR TITLE
Add GenerateSecurityAudit to Get-DbaPrivilege

### LINKED ISSUE #6173: Get-DbaPrivilege - Add SeAuditPrivilege to output
### Summary of new feature

SeAuditPrivilege ('Generate security audits') is required when you want to start writing SQL Audit logging to the Windows Security Log.

### PR DESCRIPTION
**Pull request by ben-thul**
## Type of Change

 - [ ] Bug fix (non-breaking change, fixes # )
 - [x] New feature (non-breaking change, adds functionality, fixes #6173 6173 )
 - [ ] Breaking change (effects multiple commands or functionality, fixes # )
 - [ ] Ran manual Pester test and has passed (`.\tests\manual.pester.ps1)
 - [ ] Adding code coverage to existing functionality
 - [ ] Pester test is included
 - [ ] If new file reference added for test, has is been added to github.com/sqlcollaborative/appveyor-lab ?
 - [ ] Nunit test is included
 - [ ] Documentation
 - [ ] Build system
 

### Purpose
 
Add functionality to Get-DbaPrivilege to check for SeAuditPrivilege ('Generate security audits') in the Local Security Policy
### Approach

Simple copy/paste/adapt of existing code from the same function.

### Commands to test

Get-DbaPrivilege

### CODE REVIEW

File: functions/Get-DbaPrivilege.ps1 [hunk 1]
```diff
@@ -145,14 +148,35 @@ function Get-DbaPrivilege {
...
+                }
+                $users = @() + $BL + $ifi + $lpim + $gsa | Select-Object -Unique
                 $users | ForEach-Object {
                     [PSCustomObject]@{
                         ComputerName              = $computer
                         User                      = $_
                         LogonAsBatch              = $BL -contains $_
                         InstantFileInitialization = $ifi -contains $_
                         LockPagesInMemory         = $lpim -contains $_
+                        GenerateSecurityAudit     = $gsa -contains $_
```
Comment (wsmelton) [inline functions/Get-DbaPrivilege.ps1]: Please add screenshots to show this works as I do not have any lab environment to test.

**Comment by wsmelton** [comment]
How does this setting also pertain to SQL Server...what issue/feature request is being resolved by this addition?

**Comment by ben-thul** [comment]
This permission is required in order to allow SQL Audit to log to the
Windows security log.

--
Ben

**Comment by wsmelton** [comment]
Ah, ok...never log to security log so was not aware of that one 👍

**Pull request merged**